README.md

```
@@ -4,6 +4,8 @@
 
 Smart Contracts for the HOPR mixnet.
 
+### Current addresses are placed in [`/ethereum/bindings/contracts-addresses.json`](https://github.com/hoprnet/contracts/blob/main/ethereum/bindings/contracts-addresses.json).
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)